README.md

```diff
@@ -4,6 +4,11 @@ Example repository for the blog post [Kubernetes Deep Dive: Code Generation for
 
 ## Getting Started
 
+Checkout this repo under this folder and set the GOPATH
+git clone https://github.com/openshift-evangelists/crd-code-generation.git ~/src/github.com/openshift-evangelists/crd-code-generation/
+export GOPATH=~
```

Comment:
Having a proper GOPATH is necessary while dealing with Go projects and I don't think it's necessary to include it here.

Maybe an "Installing" section would be a better home for this. And just `go get github.com/openshift-evangelists/crd-code-generation` should be enough to install it.

Reply:
Isn't there a default GOPATH now with go 1.10?

Reply:
I had it working by git checkout the source under a specific structure (~/src/github.com/openshift-evangelists/crd-code-generation/) and have the GOPATH point to the root of structure. Again this is not a problem for me anymore. But I think that if someone starts using your project that is new in GO make the barrier of entry higher than what is needed. Additional installation instruction would just make it easier for someone new to start using it.

Reply:
Can you update the PR with `~/go` as GOPATH. I think that's the most common. Also please use `go get github.com/openshift-evangelists/crd-code-generation` instead of git-clone.

Reply:
> Isn't there a default GOPATH now with go 1.10?

It was there from go 1.8 :) https://golang.org/doc/go1.8#gopath

> Can you update the PR with ~/go as GOPATH? 

What if someone already has the GOPATH set and it is not `~/go`?

Reply:
> What if someone already has the GOPATH set and it is not ~/go?

Should be formulated properly to make that clear.